lib/rollbax.ex

```diff
@@ -71,6 +71,10 @@ defmodule Rollbax do
   def start(_type, _args) do
     config = init_config()
 
+    unless Enum.member?([true, false, :log, nil], config[:enabled]) do
```

Comment:
The error message looks good. However, I would simplify the check a little bit and remove the check for `nil`. I can't recall right now, but if we don't handle `nil` we should have a default value in the application environment.

```suggestion
    unless config[:enabled] in [true, false, :log] do
```

Reply:
In a rush, I couldn't quite figure out what to do with `nil` either. Thanks for the refactor 😄